pkg/utils/kubelessutils.go

```diff
@@ -0,0 +1,31 @@
+package utils
```

Comment:
I am not sure about splitting just this function into a different file. I know that the current k8sutils.go is huge but if we split it we should it applying some criteria. For the moment I would move this function to that file and we can open a issue to properly split the file.

Reply:
This is `kubeless` specific utils. To separate concerns here, I put it in a separate file which will have utility methods only for kubeless. `k8sutils.go` contains methods which are mainly k8s specific. IMO we would need kubeless specific utils any time soon. So why not have it now?
An issue on this refactoring sounds good. `kubelessutils.go` will provide some groundwork for that.